src/Microsoft.ML.Data/DataLoadSave/LegacyCompositeDataLoader.cs

```diff
@@ -412,9 +413,11 @@ internal TransformerChain<ITransformer> GetTransformer()
                 }
                 else
                 {
-                    ITransformer transformer = new TransformWrapper(_host, transform.Transform);
+                    ITransformer transformer = new TransformWrapper(_host, transform.Transform, false, lastTransformer is RowToRowMapperTransform);
```

Comment:
>lastTransformer is RowToRowMapperTransform [](start = 103, length = 42)

Does this work? If you have two transforms in a row that are not `RowToRowMapperTransform`, wouldn't we have the same problem again of applying the whole pipeline in the `TransformWrapper`?

Reply:
As

---
In reply to: [282968970](https://github.com/dotnet/machinelearning/pull/3700#discussion_r282968970) [](ancestors = 282968970)